src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/config/NomisPrisonerApiExceptionHandler.kt

```diff
@@ -71,13 +71,14 @@ class NomisPrisonerApiExceptionHandler {
   @ExceptionHandler(MethodArgumentNotValidException::class)
   fun handleValidationErrorException(e: MethodArgumentNotValidException): ResponseEntity<ErrorResponse?>? {
     log.info("Bad request: {}", e.message)
+    val message = if (e.hasFieldErrors()) { "${e.fieldError?.field} ${e.fieldError?.defaultMessage}" } else { e.message }
```

Comment:
You end up with something like `adjustmentDays must be greater than or equal to 0`